guide/english/css/css3-nth-child-selector/index.md

```diff
@@ -49,10 +49,42 @@ For example, the following will match every 3rd anchor element:
     /* CSS goes here */
   }
 ```
+#### Selecting element with n
+The nth child selector saves you from adding class to everything in your website. You can access each element with n being the number of the position of the element.
+```html
+  	<ul class="parent">
+      <li>First<li>
+      <li>Second</li>
+     <li>Third</li>
```

Comment:
This `<li>` tag seems out of place with the rest, consider adding a space to make it even